magenta/models/rl_tuner/.gitignore

```diff
@@ -0,0 +1,4 @@
+.ipynb_checkpoints/*
```

Comment:
do we actually want to ignore all of these? It might be nice to see that you have a .ckpt file in your git tree so you can move it somewhere else.

I think the last two should be covered by our top-level .gitignore: https://github.com/tensorflow/magenta/blob/master/.gitignore